test.py

```diff
@@ -0,0 +1,25 @@
+import sqlalchemy
+from sqlalchemy import Table, Column, Integer, String, MetaData, ForeignKey
+from sqlalchemy import create_engine
+from sqlalchemy import text
+import pandas as pd
+import os
+import re
+import psycopg2
+
+##Creating engine to connect to sql database
+#create_engine(dialect+driver://username:password@host:port/database)
```

Comment:
It's generally bad practice to leave "dead code" in and I, in particular, really dislike it.

ChatGPT says:
Commented-out code is a form of dead code because it is code that is no longer being executed, even though it is still present in the source code. Commented-out code is code that has been intentionally commented out by a developer, usually as a temporary measure while testing or debugging the program. However, if this code remains commented out in the final version of the program, it can become dead code.

Commented-out code can create some of the same problems as other types of dead code, such as making the program more complex and harder to maintain. Additionally, commented-out code can be confusing for other developers who may try to understand or modify the code, thinking that it is still in use.

Therefore, it is best to remove commented-out code before releasing the final version of the program. If the code was temporarily commented out during development, the developer should either remove it entirely or use version control to keep a record of the change. Keeping a record of commented-out code can be useful for reference or future development but it should not be included in the final version of the program.

Reply:
> It's generally bad practice to leave "dead code" in and I, in particular, really dislike it.
> 
> ChatGPT says: Commented-out code is a form of dead code because it is code that is no longer being executed, even though it is still present in the source code. Commented-out code is code that has been intentionally commented out by a developer, usually as a temporary measure while testing or debugging the program. However, if this code remains commented out in the final version of the program, it can become dead code.
> 
> Commented-out code can create some of the same problems as other types of dead code, such as making the program more complex and harder to maintain. Additionally, commented-out code can be confusing for other developers who may try to understand or modify the code, thinking that it is still in use.
> 
> Therefore, it is best to remove commented-out code before releasing the final version of the program. If the code was temporarily commented out during development, the developer should either remove it entirely or use version control to keep a record of the change. Keeping a record of commented-out code can be useful for reference or future development but it should not be included in the final version of the program.

@ChasNelson1990  In this case, I was actually trying to comment the format of create_engine() so people could easily replicate it. I should have commented it better. Or is there no need for such a thing?

Reply:
There is no need, that's what documentation is for lol